src/components/production/ProductionDashboardNavigation.jsx

```diff
@@ -0,0 +1,29 @@
+import { useNavigate } from 'react-router-dom'
+
+export default function ProductionDashboardNavigation() {
+    const navigate = useNavigate();
+
```

Comment:
The ProductionDashboardNavigation component significantly enhances the new UI experience. It provides a scalable and intuitive way to introduce additional tools for product owners without being tied to the underlying API just yet. This flexibility allows us to focus on designing a dashboard that feels purpose-built for the production workflow. I’m especially excited about expanding the functionality of these pages, as the current design reinforces the distinct context and strategic direction of this feature set.